2025-02-10 18:42:44.675 | INFO     | __main__:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 18:42:44.676 | INFO     | __main__:main:61 - STARTING utils_logger.py
2025-02-10 18:42:44.676 | INFO     | __main__:log_example:52 - This is an example info message.
2025-02-10 18:42:44.676 | WARNING  | __main__:log_example:53 - This is an example warning message.
2025-02-10 18:42:44.676 | ERROR    | __main__:log_example:54 - This is an example error message.
2025-02-10 18:42:44.677 | INFO     | __main__:main:66 - View the log output at logs\project_log.log
2025-02-10 18:42:44.677 | INFO     | __main__:main:67 - EXITING utils_logger.py.
2025-02-10 18:55:17.162 | INFO     | __main__:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 18:55:17.162 | INFO     | __main__:main:61 - STARTING utils_logger.py
2025-02-10 18:55:17.163 | INFO     | __main__:log_example:52 - This is an example info message.
2025-02-10 18:55:17.163 | WARNING  | __main__:log_example:53 - This is an example warning message.
2025-02-10 18:55:17.163 | ERROR    | __main__:log_example:54 - This is an example error message.
2025-02-10 18:55:17.163 | INFO     | __main__:main:66 - View the log output at logs\project_log.log
2025-02-10 18:55:17.164 | INFO     | __main__:main:67 - EXITING utils_logger.py.
2025-02-10 20:36:48.624 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 20:36:48.625 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 20:36:48.626 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 20:36:48.626 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:36:48.627 | ERROR    | __main__:main:62 - Error during database setup: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:36:48.629 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 20:37:40.379 | INFO     | __main__:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 20:37:40.380 | INFO     | __main__:main:61 - STARTING utils_logger.py
2025-02-10 20:37:40.380 | INFO     | __main__:log_example:52 - This is an example info message.
2025-02-10 20:37:40.380 | WARNING  | __main__:log_example:53 - This is an example warning message.
2025-02-10 20:37:40.380 | ERROR    | __main__:log_example:54 - This is an example error message.
2025-02-10 20:37:40.380 | INFO     | __main__:main:66 - View the log output at logs\project_log.log
2025-02-10 20:37:40.381 | INFO     | __main__:main:67 - EXITING utils_logger.py.
2025-02-10 20:37:45.609 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 20:37:45.609 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 20:37:45.610 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 20:37:45.611 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:37:45.611 | ERROR    | __main__:main:62 - Error during database setup: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:37:45.611 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 20:53:46.730 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 20:53:46.730 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 20:53:46.731 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 20:53:46.731 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:53:46.732 | ERROR    | __main__:main:62 - Error during database setup: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:53:46.732 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 20:56:35.802 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 20:56:35.803 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 20:56:35.804 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 20:56:35.804 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:56:35.805 | ERROR    | __main__:main:62 - Error during database setup: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-10 20:56:35.805 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 21:01:05.499 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:01:05.500 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 21:01:05.501 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:01:05.502 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-10 21:01:05.511 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-10 21:01:05.517 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-10 21:01:05.518 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-10 21:01:05.518 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 21:05:57.529 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:05:57.530 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:05:57.531 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:05:57.531 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:05:57.532 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:06:08.521 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:06:08.522 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 21:06:08.523 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:06:08.532 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-10 21:06:08.540 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-10 21:06:08.548 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-10 21:06:08.548 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-10 21:06:08.550 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 21:06:12.627 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:06:12.627 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:06:12.628 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:06:12.628 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:06:12.629 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:06:58.631 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:06:58.631 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:06:58.632 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:06:58.633 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:06:58.633 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:14:04.380 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:14:04.380 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:14:04.381 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:14:04.382 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:14:04.382 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:15:33.397 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:15:33.398 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:15:33.399 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:15:33.399 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:15:33.400 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:18:18.433 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:18:18.433 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-10 21:18:18.434 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:18:18.435 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-10 21:18:18.435 | INFO     | __main__:main:63 - Database connection closed.
2025-02-10 21:21:59.249 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:21:59.250 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 21:21:59.251 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:21:59.257 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-10 21:21:59.276 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-10 21:21:59.286 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-10 21:21:59.286 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-10 21:21:59.287 | INFO     | __main__:main:65 - Database connection closed.
2025-02-10 21:22:30.667 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-10 21:22:30.667 | INFO     | __main__:main:38 - Starting database setup...
2025-02-10 21:22:30.668 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-10 21:22:30.676 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-10 21:22:30.691 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-10 21:22:30.700 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-10 21:22:30.700 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-10 21:22:30.701 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 14:50:05.008 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 14:50:05.008 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 14:50:05.010 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 14:50:05.022 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 14:50:05.037 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 14:50:05.049 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 14:50:05.051 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 14:50:05.052 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 14:50:08.864 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 14:50:08.865 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 14:50:08.866 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 14:50:08.866 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 14:50:08.866 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 15:46:32.343 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 15:46:32.343 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 15:46:32.344 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 15:46:32.345 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 15:46:32.345 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 15:46:41.729 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 15:46:41.730 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 15:46:41.731 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 15:46:41.752 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 15:46:41.764 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 15:46:41.772 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 15:46:41.772 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 15:46:41.774 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 18:05:07.215 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:05:07.216 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 18:05:07.216 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:05:07.223 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 18:05:07.229 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 18:05:07.234 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 18:05:07.234 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 18:05:07.235 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 18:05:12.186 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:05:12.186 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:05:12.187 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:05:12.187 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 18:05:12.188 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 18:27:09.334 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:27:09.334 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:27:09.335 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:27:09.335 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 18:27:09.336 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 18:27:09.336 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 18:27:09.336 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 18:30:54.039 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:30:54.040 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:30:54.040 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:30:54.041 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\delete_records.sql: sub-select returns 5 columns - expected 1
2025-02-14 18:30:54.041 | ERROR    | __main__:main:60 - Error during feature queries execution: sub-select returns 5 columns - expected 1
2025-02-14 18:30:54.041 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 18:33:12.185 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:33:12.185 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:33:12.185 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:33:12.186 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 18:33:12.187 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 18:33:12.187 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 18:33:12.187 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 18:35:56.043 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:35:56.043 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:35:56.043 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:35:56.044 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 18:35:56.045 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: no such column: last.name
2025-02-14 18:35:56.045 | ERROR    | __main__:main:60 - Error during feature queries execution: no such column: last.name
2025-02-14 18:35:56.045 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 18:42:31.116 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 18:42:31.117 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 18:42:31.117 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 18:42:31.118 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 18:42:31.119 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: no such column: last_name
2025-02-14 18:42:31.119 | ERROR    | __main__:main:60 - Error during feature queries execution: no such column: last_name
2025-02-14 18:42:31.119 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:20:19.142 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:20:19.142 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:20:19.143 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:20:19.149 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:20:19.154 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:20:19.158 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 19:20:19.158 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 19:20:19.158 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:20:22.130 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:20:22.130 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:20:22.131 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:20:22.132 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:20:22.132 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: no such column: last_name
2025-02-14 19:20:22.132 | ERROR    | __main__:main:60 - Error during feature queries execution: no such column: last_name
2025-02-14 19:20:22.132 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:21:59.529 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:21:59.529 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:21:59.530 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:21:59.530 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:21:59.531 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:21:59.531 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:21:59.531 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:22:26.070 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:22:26.070 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:22:26.071 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:22:26.072 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:22:26.072 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:22:26.072 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:22:26.073 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:22:44.422 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:22:44.422 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:22:44.423 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:22:44.424 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:22:44.424 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: unrecognized token: ":"
2025-02-14 19:22:44.424 | ERROR    | __main__:main:60 - Error during feature queries execution: unrecognized token: ":"
2025-02-14 19:22:44.425 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:23:03.589 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:23:03.589 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:23:03.590 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:23:03.591 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:23:03.591 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:23:03.591 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:23:03.592 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:23:55.769 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:23:55.769 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:23:55.770 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:23:55.771 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:23:55.772 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:23:55.772 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:23:55.772 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:24:30.365 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:24:30.366 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:24:30.366 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:24:30.368 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:24:30.368 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:24:30.368 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:24:30.369 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:25:34.019 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:25:34.020 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:25:34.021 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:25:34.022 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:25:34.022 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:25:34.022 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:25:34.023 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:25:49.751 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:25:49.751 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:25:49.751 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:25:49.753 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:25:49.753 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:25:49.754 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:25:49.754 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:26:50.622 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:26:50.623 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:26:50.623 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:26:50.624 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:26:50.625 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:26:50.625 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:26:50.625 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:31:26.414 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:31:26.415 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:31:26.415 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:31:26.416 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:31:26.417 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:31:26.417 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:31:26.417 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:35:39.199 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:35:39.199 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:35:39.200 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:35:39.200 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-14 19:35:39.200 | ERROR    | __main__:main:62 - Error during database setup: [Errno 2] No such file or directory: 'C:\\Projects\\datafun-05-sql\\sql_create\\01_drop_tables.sql'
2025-02-14 19:35:39.201 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:36:09.259 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:36:09.260 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:36:09.260 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:36:09.266 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:36:09.270 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:36:09.271 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql: table authors has no column named author_name
2025-02-14 19:36:09.271 | ERROR    | __main__:main:62 - Error during database setup: table authors has no column named author_name
2025-02-14 19:36:09.271 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:36:35.972 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:36:35.973 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:36:35.973 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:36:35.980 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:36:35.985 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:36:35.985 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql: table authors has no column named author
2025-02-14 19:36:35.986 | ERROR    | __main__:main:62 - Error during database setup: table authors has no column named author
2025-02-14 19:36:35.986 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:37:09.654 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:37:09.654 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:37:09.655 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:37:09.662 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:37:09.666 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:37:09.667 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql: table authors has no column named first
2025-02-14 19:37:09.667 | ERROR    | __main__:main:62 - Error during database setup: table authors has no column named first
2025-02-14 19:37:09.667 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:38:17.491 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:38:17.492 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:38:17.493 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:38:17.499 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:38:17.503 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:38:17.503 | ERROR    | __main__:execute_sql_file:32 - Failed to execute C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql: table authors has no column named author_name
2025-02-14 19:38:17.504 | ERROR    | __main__:main:62 - Error during database setup: table authors has no column named author_name
2025-02-14 19:38:17.504 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:39:02.789 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:39:02.789 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:39:02.790 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:39:02.797 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:39:02.800 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:39:02.804 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 19:39:02.804 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 19:39:02.804 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:39:08.322 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:39:08.322 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:39:08.323 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:39:08.324 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:39:08.324 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:39:08.324 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:39:08.325 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:40:14.584 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:40:14.584 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:40:14.585 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:40:14.586 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:40:14.586 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:40:14.586 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:40:14.586 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:40:28.461 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:40:28.461 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:40:28.462 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:40:28.463 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:40:28.463 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:40:28.463 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:40:28.464 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:40:38.627 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:40:38.627 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:40:38.628 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:40:38.629 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:40:38.629 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:40:38.629 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:40:38.629 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:40:48.831 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:40:48.831 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:40:48.832 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:40:48.833 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:40:48.833 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:40:48.833 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:40:48.833 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:40:58.285 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:40:58.286 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:40:58.286 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:40:58.287 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:40:58.288 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: near ".": syntax error
2025-02-14 19:40:58.288 | ERROR    | __main__:main:60 - Error during feature queries execution: near ".": syntax error
2025-02-14 19:40:58.288 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:41:30.706 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:41:30.706 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:41:30.707 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:41:30.708 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:41:30.708 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 19:41:30.708 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 19:41:30.709 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:48:36.045 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:48:36.046 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:48:36.046 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:48:36.053 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:48:36.061 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:48:36.066 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 19:48:36.066 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 19:48:36.066 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:48:40.396 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:48:40.397 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:48:40.397 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:48:40.398 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:48:40.398 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 19:48:40.399 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 19:48:40.399 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:54:48.259 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:54:48.259 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:54:48.260 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:54:48.266 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:54:48.270 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:54:48.273 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 19:54:48.273 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 19:54:48.273 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:54:51.201 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:54:51.201 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:54:51.202 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:54:51.203 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:54:51.203 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 19:54:51.203 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 19:54:51.204 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 19:56:51.330 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:56:51.330 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 19:56:51.331 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:56:51.337 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 19:56:51.343 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 19:56:51.347 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 19:56:51.348 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 19:56:51.348 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 19:56:54.007 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 19:56:54.007 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 19:56:54.008 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 19:56:54.009 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 19:56:54.009 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 19:56:54.010 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 19:56:54.010 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 20:01:55.130 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:01:55.131 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 20:01:55.131 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:01:55.138 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 20:01:55.141 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 20:01:55.145 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 20:01:55.145 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 20:01:55.145 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 20:03:13.923 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:03:13.923 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 20:03:13.924 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:03:13.925 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 20:03:13.925 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 20:03:13.926 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 20:03:13.926 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 20:06:08.929 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:06:08.929 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 20:06:08.930 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:06:08.931 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 20:06:08.932 | ERROR    | __main__:execute_sql_file:29 - Failed to execute C:\Projects\datafun-05-sql\sql_features\update_records.sql: duplicate column name: birth_year
2025-02-14 20:06:08.932 | ERROR    | __main__:main:60 - Error during feature queries execution: duplicate column name: birth_year
2025-02-14 20:06:08.932 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 20:06:28.218 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:06:28.219 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 20:06:28.219 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:06:28.220 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 20:06:28.220 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 20:06:28.220 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 20:06:28.221 | INFO     | __main__:main:63 - Database connection closed.
2025-02-14 20:22:01.569 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:22:01.570 | INFO     | __main__:main:38 - Starting database setup...
2025-02-14 20:22:01.571 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:22:01.576 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\01_drop_tables.sql
2025-02-14 20:22:01.581 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\02_create_tables.sql
2025-02-14 20:22:01.586 | INFO     | __main__:execute_sql_file:30 - Executed: C:\Projects\datafun-05-sql\sql_create\03_insert_records.sql
2025-02-14 20:22:01.586 | INFO     | __main__:main:60 - Database setup completed successfully.
2025-02-14 20:22:01.586 | INFO     | __main__:main:65 - Database connection closed.
2025-02-14 20:22:05.224 | INFO     | utils_logger:<module>:39 - Logging to file: logs\project_log.log
2025-02-14 20:22:05.225 | INFO     | __main__:main:36 - Starting feature queries execution...
2025-02-14 20:22:05.225 | INFO     | __main__:main:52 - Connected to database: C:\Projects\datafun-05-sql\data\db.sqlite
2025-02-14 20:22:05.226 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\delete_records.sql
2025-02-14 20:22:05.227 | INFO     | __main__:execute_sql_file:27 - Executed: C:\Projects\datafun-05-sql\sql_features\update_records.sql
2025-02-14 20:22:05.227 | INFO     | __main__:main:58 - Feature queries execution completed successfully.
2025-02-14 20:22:05.227 | INFO     | __main__:main:63 - Database connection closed.
